law_system/law_vectorstore.py

```
@@ -175,25 +193,36 @@ def create_vectorstore(self, documents: List[Document], chunk_size: int = 1000,
             chunk_overlap: 청크 겹침 크기
         """
         if not documents:
+            logger.warning("⚠️ 문서가 없어 벡터스토어를 생성할 수 없습니다.")
             return
 
+        logger.info(f"\n🔨 벡터스토어 생성 시작")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)